src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACYTY = 16;
+    private static final int GROW_CONSTANT = 2;
+    private static final double GROWE_INDEX = 0.75;
```

Comment:
```suggestion
    private static final double GROWE_INDEX = 0.75;
```
please use more appropriate naming here